hack/vendor.sh

```diff
@@ -12,4 +12,17 @@ if ! hash vndr; then
 	exit 1
 fi
 
-vndr -whitelist=^archive/tar "$@"
+if [ $# -eq 0 ] || [ "$1" = "archive/tar" ]; then
+	echo "update vendored copy of archive/tar"
+	: "${GO_VERSION:=$(awk -F '[ =]' '$1 == "ARG" && $2 == "GO_VERSION" { print $3; exit }' ./Dockerfile)}"
+	rm -rf vendor/archive
+	mkdir -p ./vendor/archive/tar
+	echo "downloading: https://golang.org/dl/go${GO_VERSION}.src.tar.gz"
+	curl -fsSL "https://golang.org/dl/go${GO_VERSION}.src.tar.gz" \
+		| tar --extract --gzip --directory=vendor/archive/tar --strip-components=4 go/src/archive/tar
+	patch --strip=4 --directory=vendor/archive/tar --input="$PWD/patches/0001-archive-tar-do-not-populate-user-group-names.patch"
```

Comment:
Is the tar patch still relevant with this new go version?

Reply:
Yes, it's the `user.LookupId`  / `user.LookupGroupId` that we can't have, related to https://nvd.nist.gov/vuln/detail/CVE-2019-14271